classes/Domain/Services/TalkEmailer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Domain\Services;
+
+use Swift_Mailer;
+use Swift_Message;
+
+class TalkEmailer
```

Comment:
Can we mark this class as `final`? That is, unless we intend it not to be `final` so other people can extend it.